pkg/container/kubernetes/client.go

```diff
@@ -806,8 +806,10 @@ func (c *Client) createHeadlessService(
 		}
 	}
 
+	svcName := fmt.Sprintf("%s-headless", containerName)
```

Comment:
should we do something more explicit? So we don't forget. Something like `<MCP name>-proxy` or `<MCP name>-proxy-headless` or something of the sort?

Reply:
Yeah, shall I rename the one created by the controller also, just to stay consistent?
`mcp-proxy`
`mcp-server-headless`